index.d.ts

```diff
@@ -20,6 +20,11 @@ declare namespace fastifySwagger {
   interface FastifyDynamicSwaggerOptions extends FastifySwaggerOptions {
     mode?: 'dynamic';
     swagger?: Partial<SwaggerSchema.Spec>;
+    /**
+     * Overwrite the route schema
+     * @default false
+     */
```

Comment:
I don’t think the default is false. Typings also have tests.

Reply:
Yeah, sorry I'm not a heavy user of TypeScript, I'm gonna fix it and currently looking at for the unit-tests

Reply:
Where do you usually test dynamic mode ?

Reply:
Sorry for being so long about the unit-tests, I'm not familiar with `tap`.
Also I don't know to which extend you need to test the property.